### PR TITLE
Some proofreading for the Spring Web guide

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

File: docs/src/main/asciidoc/spring-web-guide.adoc [hunk 1]
```diff
@@ -153,14 +153,14 @@ Open your browser to http://localhost:8080/greeting/world.
...
 The result should be: `{"message": "HELLO WORLD!"}`.
 
-== Run the application as a native
+== Run the application as a native executable
 
-You can of course create a native image using instructions similar to link:building-native-image-guide.html[this] guide.
+You can of course create a native image using the instructions of the link:building-native-image-guide.html[Building a native executable guide].
 
-== Supported Spring Web functionality
+== Supported Spring Web functionalities
```
Comment (geoand) [inline docs/src/main/asciidoc/spring-web-guide.adoc]: I'm not sure about this one, I personally wouldn't use the plural form, but if you think it's better, fine by me

File: docs/src/main/asciidoc/spring-web-guide.adoc [hunk 2]
```diff
@@ -153,14 +153,14 @@ Open your browser to http://localhost:8080/greeting/world.
...
 
-Quarkus currently supports a subset of the functionality that Spring Web provides. More specifically Quarkus supports the REST related features of Spring Web
-(in a high level, think of `@RestController` instead of `@Controller`).
+Quarkus currently supports a subset of the functionalities that Spring Web provides. More specifically Quarkus supports the REST related features of Spring Web
```
Comment (geoand) [inline docs/src/main/asciidoc/spring-web-guide.adoc]: Same here

**Review by geoand — APPROVED** [review]

File: docs/src/main/asciidoc/spring-web-guide.adoc (hunk 1 above)
Comment (gsmet) [reply docs/src/main/asciidoc/spring-web-guide.adoc]: I checked here before adding the comment: https://www.merriam-webster.com/dictionary/functionality and they have an example saying "data management functionalities such as data integrity, security, recoverability and manageability".

I also found "The software update adds interesting functionalities." here https://www.linguee.fr/anglais-francais/traduction/functionality.html .

We are talking about the supported features so plural it should be IMHO.

Comment (geoand) [reply docs/src/main/asciidoc/spring-web-guide.adoc]: Cool!

**Pull request merged**